cogs5e/sheetManager.py

```diff
@@ -919,6 +919,11 @@ async def import_sheet(self, ctx, url: str, *, args=""):
         if character and beyond_match:
             await send_ddb_ctas(ctx, character)
 
+        # Update charsetting based on the version argument
+        character.options.version = version
+        await character.options.commit(ctx.bot.mdb, character)
```

Comment:
Not sure how early this is, but this probably should be in `SheetManager._load_sheet` so that you don't have to commit twice.

Reply:
I think it is good to put it in ._load_sheet, but I have to commit twice though:
1. character.commit for the character model
2. character.options.commit for the character settings.

@SeanStoves Does it makes sense?

Reply:
> I think it is good to put it in ._load_sheet, but I have to commit twice though:
> 
> 1. character.commit for the character model
> 2. character.options.commit for the character settings.
> 
> @SeanStoves Does it makes sense?

@kevklash I do see the point of putting it in ``_load_sheet``.   If we put it in that function, it will be good across the board.  However, I'm concerned about passing the version into that. and how further uses of it will be.

Reply:
`character.options` is included in a `Character.commit` based on `Character.to_dict`, it seems that `CharacterSettings.commit` is just a convenience method for when you don't need to commit the whole character, however in this case we need to commit the whole character anyways so might as well.

Reply:
> `character.options` is included in a `Character.commit` based on `Character.to_dict`. It seems that `CharacterSettings.commit` is just a convenience method for when you don't need to commit the whole character, however in this case we need to commit the whole character anyways so might as well.

Sounds good.. @kevklash Let's give that a whirl in that function.   We should be solid ;)

Reply:
Also might I suggest using a command argument like `-version` to specify a version, rather than a positional argument? We're already passing `args` to `_load_sheet` so you don't actually need to change the function signature for either `import` (compared to master) or `_load_sheet` to make this change if done this way.

Reply:
@SeanStoves @LazyDope Noted! Making changes and testing locally now.